pkg/sources/github/receive_adapter/receive_adapter.go

```diff
@@ -54,8 +57,26 @@ type GithubSecrets struct {
 // HandlePullRequest is invoked whenever a PullRequest is modified (created, updated, etc.)
 func (h *GithubHandler) HandlePullRequest(payload interface{}, header webhooks.Header) {
 	log.Print("Handling Pull Request")
-	pl := payload.(github.PullRequestPayload)
-	postMessage(h.target, &pl)
+
+	hdr := http.Header(header)
+
+	pl := payload.(gh.PullRequestPayload)
+
+	source := pl.PullRequest.HTMLURL
+
+	eventType, ok := github.CloudEventType[hdr.Get("X-GitHub-Event")]
```

Comment:
`github.CloudEventType` should map "" to `UnsupportedEvent`

Reply:
It does, this catches all other github events that have a real name but are not "" and "pull_request"

Reply:
Ah, right. I was thinking about the "header missing" case.